code/modules/modular_computers/computers/item/computer_ui.dm

```diff
@@ -13,6 +13,11 @@
 			ui.close()
 		return
 
+	if(HAS_TRAIT(user, TRAIT_CHUNKYFINGERS))
```

Comment:
1. Why does this close the UI? This should just make it interactive only.
2. This needs feedback (which is going to be harder with #1, since `ui_status` should be pure).

Reply:
also, will this screw over free golems and the like?

Reply:
i removed the ui.close(), is interact only now. i also added a small feedback to it